test/webaudio/audioworklet_worker.c

```diff
@@ -13,17 +11,19 @@
 int workletToWorkerFutexLocation = 0;
 int workletToWorkerFlag = 0;
 
+void do_exit() {
+  emscripten_terminate_all_wasm_workers();
+  emscripten_force_exit(EXIT_SUCCESS);
```

Comment:
We normally just use `0` in test code rather than `EXIT_SUCCESS`

Reply:
I'm ancient and proper C is `EXIT_SUCCESS`. But I'm not fussy.